src/parser.coffee

```diff
@@ -83,6 +83,9 @@ class VASTParser
             if node.nodeName is 'Ad'
                 ad = @parseAdElement node
                 if ad?
+                    if options.determinedSequence?
+                        ad.sequence = options.determinedSequence
```

Comment:
@eliamaino-fp Sorry for the late reply.

I would think that if there is a sequence on the wrapped ad it should take priority over what's on the wrapper, so definitely. To be safe I can only set the sequence if it doesn't exist on the ad already, does that sound good?

Since the sequence is set on the options on each parse and just before resolving wrapped ads it'll carry over through a long chain and won't be set if there isn't one on the wrapper.

Reply:
Do you have any reference in the VAST specification which states that? What if a wrapper with a `sequence` attribute contains a pod of ads each one with a personal `sequence` attribute?
(this may be possible accordng to section 2.3.5.2 of the spec)

Reply:
@eliamaino-fp Makes sense - I'll update the pr to only use the wrapper sequence if it doesn't exists on the wrapped ad already